deploy/build_qa_artifact

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+DEPLOY_ID=$1
+
+set -e
+set -x
+
+mv .fakenpmrc .npmrc
+
+cp docker-compose.yml docker-compose-ci.yml
+
+prepare-docker-compose --directory pyxis-npm && cp docker-compose-qainit.yml docker-compose.yml
+
+docker-compose build web
+docker-compose run -w $PWD -u root -e NPM_TOKEN=$NPM_TOKEN --entrypoint /bin/sh web \
+'-c' 'npm view prima-assicurazioni/pyxis-npm versions --json > versions.json'
+
+get_pyxis_version=`sed -E 's/[A-Za-z|^0]*//g' <<<"$DEPLOY_ID" | cut -c1-10`
+qa_versions=`cat versions.json | jq --arg pyxis_version "$get_pyxis_version" '. | contains([$pyxis_version])'`
+pyxis_version="0.${get_pyxis_version}.${#qa_versions[@]}"
```

Comment:
```suggestion
pyxis_version="0.${get_pyxis_version}.${qa_versions[@]}"
```

Reply:
lo rimetto, perché # serve in bash proprio per avere la dimensione di qa_versions, non è un refuso preso da twig

Reply:
Ah sorry, non conoscevo questa sintassi di bash. Ottimo :+1: